recipes/bitserializer/all/conanfile.py

```diff
@@ -52,14 +57,29 @@ def requirements(self):
         if self.options.with_cpprestsdk:
             self.requires("cpprestsdk/2.10.18", transitive_headers=True, transitive_libs=True)
         if self.options.with_rapidjson:
-            self.requires("rapidjson/cci.20220514", transitive_headers=True, transitive_libs=True)
+            self.requires("rapidjson/cci.20220822", transitive_headers=True, transitive_libs=True)
         if self.options.with_pugixml:
             self.requires("pugixml/1.13", transitive_headers=True, transitive_libs=True)
+        if self.options.with_rapidyaml:
+            self.requires("rapidyaml/0.4.1", transitive_headers=True, transitive_libs=True)
 
     def package_id(self):
         self.info.clear()
 
     def validate(self):
+        # ToDo: remove this after fix linkage the base library
+        if self.options.with_rapidyaml:
+            raise ConanInvalidConfiguration("YAML currently unavailable due an issue with linking the base library")
```

Comment:
Is there any issue link which could be associated? The message is too generic and we need a north to know when removing it.

Reply:
I will create separate issue for this, this exception happens only when link `RapidYaml` library, this step passes if replace for example to "PugiXml", but I don't see any suspicioos in the `RapidYaml` recipe.

Reply:
I've created issue: #14972

Reply:
It's about test package, it should not be a blocker. I'll take a look on that issue, thanks for opening it.